packages/create-plugin/src/commands/generate.plopfile.ts

```diff
@@ -67,6 +67,12 @@ export default function (plop: NodePlopAPI) {
         default: false,
         when: (answers: CliArgs) => answers.pluginType !== PLUGIN_TYPES.panel,
       },
+      {
```

Comment:
I'm cautious of adding yet another question, and also concerned that this is not something which is going to be intuitive to new plugin developers. 

Is this something which can be added safely later?
Are there negatives to just adding it without asking?

Reply:
Yeah, after thinking about it for a bit, I agree with @sympatheticmoose, we start to have too many questions, and I am afraid we are already in the state where folks just "Enter their way through". 

Let's remove the prompt for this for now, and maybe add a README file under `templates/provisioning-template` in the meantime, which explains a bit why the provisioning files can be useful, and also has some links for more information on how to work with them.

Reply:
+1. I feel we already ask too many questions. I was even thinking on removing the questions about workflows and always generating them.